.env.example

```
@@ -0,0 +1,4 @@
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+DATABASE_URL=
```

Думаю, здесь никаких секретов нет, вполне типовые значения для локалки. Тем не менее без их указания может быть не очень понятно, а что же должно быть в `.env`.

```suggestion
POSTGRES_DB=wheremylikes_development
POSTGRES_USER=postgres
POSTGRES_PASSWORD=postgres
DATABASE_URL=postgresql://postgres@postgres:5432
```